packages/runtime/runtime.ts

```diff
@@ -112,6 +113,7 @@ export interface Runtime {
   showNotification(data: DcNotification): void
   clearAllNotifications(): void
   clearNotifications(chatId: number): void
+  // enables to set a callback (used in frontend RuntimeAdapter)
```

Comment:
```suggestion
  /** enables to set a callback (used in frontend RuntimeAdapter) */
```

documentation should use a doc comment so it is shown when you hover on it in the IDE